app/ocr.py

```diff
@@ -19,7 +19,9 @@
 from spacy.matcher import Matcher, PhraseMatcher
 from typing import List, Tuple, Union, Callable, Dict, Iterator
 
-nlp = load("en_core_web_md")
+# change to small
+# nlp = load("en_core_web_md")

```

Comment:
Thank you for documenting this change. Could you remove the commented our code so this is not left in the codebase?

Reply:
Good eyes.  Will do, Dylan!

Reply:
Thanks Frank! I believe this is also where we are getting that merge conflict. The "from typing" import and loading the "en_core_web_sm" from spacy are conflicting with main. Everything looks good, just have to resolve those conflicts before this branch is able to merge.